examples/ruby/spec/getting_started/first_script.rb

```diff
@@ -18,3 +18,6 @@
 message.text
 
 driver.quit
+
+# To run this script, execute the following command:
+# ruby first_script.rb
```

Comment:
You can add those comments [here](https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/ruby/README.md). In a generic way, ideally.